### ISSUE TITLE
Issue when generating certificate with certbot

### ISSUE THREAD

**Issue by mabasic** [plausible/hosting#22]
I am following the documentation [here](https://github.com/plausible/hosting/tree/master/reverse-proxy#nginx), but when I enter:

```
sudo certbot --nginx
```

and select the virtual host, I get this error:

```
Obtaining a new certificate
Performing the following challenges:
http-01 challenge for plausible.laravelista.com
Waiting for verification...
Cleaning up challenges
Failed authorization procedure. plausible.laravelista.com (http-01): urn:ietf:params:acme:error:unauthorized :: The client lacks sufficient authorization :: Invalid response from http://plausible.laravelista.com/.well-known/acme-challenge/RIILIq6-u-mzcqo7mT8hDesS-VfPn1O1-obtFGt7ijI [2a03:b0c0:3:e0::476:7001]: "<html>\r\n<head><title>404 Not Found</title></head>\r\n<body>\r\n<center><h1>404 Not Found</h1></center>\r\n<hr><center>nginx/1.16.1</ce"

IMPORTANT NOTES:
 - The following errors were reported by the server:

   Domain: plausible.laravelista.com
   Type:   unauthorized
   Detail: Invalid response from
   http://plausible.laravelista.com/.well-known/acme-challenge/RIILIq6-u-mzcqo7mT8hDesS-VfPn1O1-obtFGt7ijI
   [2a03:b0c0:3:e0::476:7001]: "<html>\r\n<head><title>404 Not
   Found</title></head>\r\n<body>\r\n<center><h1>404 Not
   Found</h1></center>\r\n<hr><center>nginx/1.16.1</ce"

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A/AAAA record(s) for that domain
   contain(s) the right IP address.
```

For some reason certbot is not getting the response from plausible container, but from my local nginx. When I try manually visiting the URL in browser I get the 404 response from plausible container.

Am I missing some options in the nginx virtual host config?

plausible.cnf

```
server {
        server_name plausible.laravelista.com;

        location / {
                proxy_pass http://127.0.0.1:8000;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        }
}
```

**Comment by mabasic** [comment]
I have found a fix for this. I'll send a PR soon.

**Comment by ukutaht** [comment]
Fixed in #23
